### ISSUE TITLE
ParseError when getting dividends data for certain tickers

### ISSUE THREAD

**Issue by ymyke** [alvarobartt/investpy#137]
Trying to retrieve dividend data for certain stocks results in a `ParseError`.

E.g., this: `investpy.get_stock_dividends("UBSG", "switzerland")`

Results in:

```
~\Anaconda3\envs\fignal\lib\site-packages\investpy\stocks.py in get_stock_dividends(stock, country)
    939                     flag = False
    940 
--> 941                 root_ = fromstring(res['historyRows'])
    942                 path_ = root_.xpath(".//tr")
    943 

~\Anaconda3\envs\fignal\lib\site-packages\lxml\html\__init__.py in fromstring(html, base_url, parser, **kw)
    873     else:
    874         is_full_html = _looks_like_full_html_unicode(html)
--> 875     doc = document_fromstring(html, parser=parser, base_url=base_url, **kw)
    876     if is_full_html:
    877         return doc

~\Anaconda3\envs\fignal\lib\site-packages\lxml\html\__init__.py in document_fromstring(html, parser, ensure_head_body, **kw)
    762     if value is None:
    763         raise etree.ParserError(
--> 764             "Document is empty")
    765     if ensure_head_body and value.find('head') is None:
    766         value.insert(0, Element('head'))

ParserError: Document is empty
```

Even though there is dividend data available on investing.com:
https://www.investing.com/equities/ubs-group-n-dividends

**Comment by alvarobartt** [comment]
Issue has been solved in 7dcc90fc136b4744f977286cee4f47847f709199, since the error reported in this issue is also related to the one reported in #131.

**Comment by alvarobartt** [comment]
__Note that currently it is just available on the developer version__ which you can install with the following command:
```
pip install git+https://github.com/alvarobartt/investpy.git@developer
```

__Thank you! Remember to star the repo if you found it useful and follow me at GitHub so as to get notified of all the updates!__
